src/datasources/asset/__snapshots__/AssetDataSource.test.ts.snap

```diff
@@ -0,0 +1,146 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`queries run metadata query 1`] = `
+[
+  {
+    "fields": [
+      {
+        "name": "id",
+        "values": [
+          "7f6d0d74-bc75-4d78-9edd-00c253b3a0de",
+          "71fbef61-fae6-4364-82d4-29feb79c7146",
+          "cf1ac843-06a2-4713-ab43-f9d1d8dfdc32",
+          "456e8812-1da4-4818-afab-f0cd34f74567",
+        ],
+      },
+      {
+        "name": "name",
+        "values": [
+          "NI-sbRIO-9629-01FE20D1",
+          "Conn0_AI",
+          "Conn0_AO",
+          "Conn0_DIO0-3",
+        ],
+      },
+      {
+        "name": "model name",
+        "values": [
+          "sbRIO-9629",
+          "AI 0-15",
+          "AO 0-3",
+          "DIO 0-3",
+        ],
+      },
+      {
+        "name": "serial number",
+        "values": [
+          "01FE20D1",
+          "01FE20D1",
+          "01FE20D1",
+          "01FE20D1",
+        ],
+      },
+      {
+        "name": "bus type",
+        "values": [
+          "BUILT_IN_SYSTEM",
+          "CRIO",
+          "CRIO",
+          "CRIO",
+        ],
+      },
+      {
+        "name": "asset type",
+        "values": [
+          "SYSTEM",
+          "GENERIC",
+          "GENERIC",
+          "GENERIC",
+        ],
+      },
+      {
+        "name": "is NI asset",
+        "values": [
+          true,
+          true,
+          true,
+          true,
+        ],
+      },
+      {
+        "name": "calibration status",
+        "values": [
+          "OK",
+          "OK",
+          "OK",
+          "OK",
+        ],
+      },
+      {
+        "name": "is system controller",
+        "values": [
+          true,
+          false,
+          false,
+          false,
+        ],
+      },
+      {
+        "name": "last updated timestamp",
+        "values": [
+          "2024-02-21T12:54:20.069Z",
+          "2024-02-21T12:54:20.072Z",
+          "2024-02-21T12:54:20.076Z",
+          "2024-02-21T12:54:20.078Z",
+        ],
+      },
+      {
+        "name": "minionId",
+        "values": [
+          "NI_sbRIO-9629--SN-01FE20D1--MAC-00-80-2F-33-30-18",
+          "NI_sbRIO-9629--SN-01FE20D1--MAC-00-80-2F-33-30-18",
+          "NI_sbRIO-9629--SN-01FE20D1--MAC-00-80-2F-33-30-18",
+          "NI_sbRIO-9629--SN-01FE20D1--MAC-00-80-2F-33-30-18",
+        ],
+      },
+      {
+        "name": "parent name",
+        "values": [
+          "",
+          "sbRIO1",
+          "sbRIO1",
+          "sbRIO1",
+        ],
+      },
+      {
+        "name": "system name",
+        "values": [
+          "NI-sbRIO-9629-01FE20D1",
+          "NI-sbRIO-9629-01FE20D1",
+          "NI-sbRIO-9629-01FE20D1",
+          "NI-sbRIO-9629-01FE20D1",
+        ],
+      },
+      {
+        "name": "calibration due date",
```

Comment:
Is this something we want to keep here ? It has 4 null values.

Reply:
taking into account this comment (https://github.com/ni/systemlink-grafana-plugins/pull/62#discussion_r1654551205), I will remove that part